src/main/java/org/schemaspy/output/diagram/OrphanDiagram.java

```diff
@@ -0,0 +1,23 @@
+package org.schemaspy.output.diagram;
+
+import java.io.File;
+
+public class OrphanDiagram {
```

Comment:
This is not used so can be removed

Reply:
On closer inspection, this class is actually used. `MustacheOrphanDiagramFactory` wasn't changed in above-mentioned PR.

Reply:
Sorry `MustacheOrphanDiagramFactory` isn't used either.

Reply:
Removing it as part of this PR would make the git history more difficult to understand. I suggest we keep the current scope, merge and create a new PR to remove the dead code.